api/kyverno/v1/spec_types.go

```diff
@@ -48,10 +48,10 @@ type Spec struct {
 	// ValidationFailureAction defines if a validation policy rule violation should block
 	// the admission review request (enforce), or allow (audit) the admission review request
 	// and report an error in a policy report. Optional.
-	// Allowed values are audit or enforce. The default value is "audit".
+	// Allowed values are audit or enforce.
+	// The default value is "" which means policy will be executed in background-only mode.
```

Comment:
can't we add another value say `background` or `background-only` instead of an empty string?

Reply:
@ApsTomar @samkulkarni20 can you suggest what would be better value for `ValidationFailureAction`?

Reply:
i confirm this with chipzoller. please check upstream issue thread.